index.html

```diff
@@ -204,7 +204,8 @@
             </li>
           </ul>
         </nav>
-
+        
+        
```

Comment:
nit: extra line

Reply:
Removed and a made another commit to this pr just now